aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/TypeConverter.java

```diff
@@ -64,6 +64,7 @@ private TypeConverter() {
         JAVA_TO_SQL.put(JavaFieldType.DATE, SQLiteDataType.TEXT);
         JAVA_TO_SQL.put(JavaFieldType.DATE_TIME, SQLiteDataType.TEXT);
         JAVA_TO_SQL.put(JavaFieldType.TIME, SQLiteDataType.TEXT);
+        JAVA_TO_SQL.put(JavaFieldType.TIMESTAMP, SQLiteDataType.INTEGER);
```

Comment:
Just to double check, Integer is correct vs. Long or Float?

Reply:
Yep, this is correct.  SQLite doesn't have different types for an int vs a long, there is only `INTEGER`. It can hold up to 64 bits, which is the same as a Java `long`, so it works for either.